Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.fedoraproject.org/fedora:31
+
+WORKDIR /tmp/mantle
+COPY . /tmp/mantle
+
+RUN yum update -y --nodocs && \
+    yum install -y --nodocs make golang gcc && \
```

Comment:
Multi-stage builds are the better way to do this.  Also, we should consider using the pre-built OpenShift golang builder images.